olive/passes/onnx/pre_post_processing.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+from typing import Any, Callable, Dict, List
+
+from onnxruntime import __version__ as OrtVersion
+
+from olive.model import ONNXModel
+from olive.passes import Pass
+from olive.passes.pass_config import PassConfigParam
+
+
+class PrePostProcessing(Pass):
```

Comment:
This name is kinda confusing me. It sounds like a general pre/postprocessing steps. Can you rename it to something like `ProcessingOnnxOps` or another meaningful name?

Reply:
Renamed to AppendPrePostProcessingOps